packages/components/src/internal/components/heatmap/HeatMapDisplay.tsx

```diff
@@ -117,133 +118,127 @@ export class HeatMapDisplay extends React.Component<HeatMapDisplayProps, any> {
             headers: xLabelsSorted,
```

Comment:
nit: No need to return `Immutable` objects here since they are just used internally and destructured during rendering. These can be consolidated into the following types:

```ts
interface DisplayCell {
    onClick?: () => void;
    style?: CSSProperties;
    title?: string;
    value: any;
}

interface DisplayData {
    headers: string[];
    rows: DisplayCell[][];
}
```

Reply:
Looks great